### ISSUE TITLE
doc bugs missing v in the tag portion of the image

### ISSUE THREAD

**Issue by krmayankk** [kubernetes/kubernetes#24778]
http://kubernetes.io/docs/getting-started-guides/docker/

in the docker run command documented above :- 

docker run \
    --volume=/:/rootfs:ro \
    --volume=/sys:/sys:ro \
    --volume=/var/lib/docker/:/var/lib/docker:rw \
    --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
    --volume=/var/run:/var/run:rw \
    --net=host \
    --pid=host \
    --privileged=true \
    --name=kubelet \
    -d \
    gcr.io/google_containers/hyperkube-amd64:${K8S_VERSION} \
    /hyperkube kubelet \
        --containerized \
        --hostname-override="127.0.0.1" \
        --address="0.0.0.0" \
        --api-servers=http://localhost:8080 \
        --config=/etc/kubernetes/manifests \
        --cluster-dns=10.0.0.10 \
        --cluster-domain=cluster.local \
        --allow-privileged=true --v=2

the image should be   gcr.io/google_containers/hyperkube-amd64:v${K8S_VERSION} \

the documentation is missing 'v' before ${K8S_VERSION}

**Comment by mikedanese** [comment]
cc @johndmulhausen

**Comment by bgrant0607** [comment]
PR welcome, @krmayankk

**Comment by luxas** [comment]
> Set the ${K8S_VERSION} variable to a released version of Kubernetes >= **“v1.2.0”**

It's working as-is, but is a breaking change from earlier, when `K8S_VERSION` was `1.2.0`. Now it should be `v1.2.0`

Reason: It's better to have the full release tag, and `latest.txt` and `stable.txt` are `v1.x.y`
```
export K8S_VERSION=$(curl -sS https://storage.googleapis.com/kubernetes-release/release/latest.txt)
```

Closing as working as intended. Reopen if you do not agree.
